server/index.js

```diff
@@ -1,30 +1,31 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const Promise = require('bluebird');
 const db = require('../database/index.js');
-let app = express();
+
+
+const app = express();
 
 
 app.use(bodyParser.json());

```

Comment:
You shouldn't need `bodyparser` here - there are no `POST` requests to retrieve a body from.